lib/users.test.js

```diff
@@ -1,3 +1,12 @@
+jest.mock('./log')
+const logGen = require('./log')
+const logger = {
+    info: jest.fn(),
+    error: jest.fn()
+}
+logGen.mockReturnValue(logger)
+const {signUp, logIn, setDBPassword} = require('./users')
```

Comment:
Shouldn't `users = require('./users')` and `const {signUp, logIn, setDBPassword} = users` be on separate lines?